java/client/src/main/java/glide/api/BaseClient.java

```diff
@@ -2692,4 +2699,119 @@ public CompletableFuture<Long> sortStore(@NonNull String key, @NonNull String de
         return commandManager.submitNewCommand(
                 Sort, new String[] {key, STORE_COMMAND_STRING, destination}, this::handleLongResponse);
     }
+
+    @Override
+    public CompletableFuture<String[]> geosearch(
+            @NonNull String key,
```

Comment:
We should also have API for GlideString to support binary data.
This comment is applicable to all new commands

Reply:
So for each method signature using `String`, a `GlideString` counterpart should be added?

Reply:
Hello @yulazariy , The agreement with Asaf was that AWS team implements all binary data transport. Given the tight deadlines, we can't take on any additional work.

Reply:
I confirm, the team don't have any spare cycles, we will take it.